src/main/resources/maps.xml

```diff
@@ -33995,4 +34004,4 @@
         <max-zoom>14</max-zoom>
         <bounds max-lat="8.277867" max-lon="-60.476261" min-lat="6.1765" min-lon="-62.532255" />
     </entry>
-</imagery>
\ No newline at end of file
+</imagery>
```

Comment:
Please don't mix code format changes with code changes. Will merge it anyway this time... :-)